docs/src/pages/guides/interoperability/StyledComponentsTheme.js

```diff
@@ -1,47 +1,36 @@
 import * as React from 'react';
-import styled, { ThemeProvider } from 'styled-components';
-import NoSsr from '@material-ui/core/NoSsr';
 import {
   createMuiTheme,
+  experimentalStyled as styled,
   ThemeProvider as MuiThemeProvider,
```

Comment:
Aliasing no longer needed?

```suggestion
  ThemeProvider,
```